docs/stacks-guide-aws.md

```diff
@@ -1,7 +1,7 @@
 ---
-title: "Crossplane Stacks Guide: AWS Setup"
+title: "AWS Stack Setup"
```

Comment:
This and other Stacks Guide changes are likely superseded by / overlap with the changes in https://github.com/crossplaneio/crossplane/pull/817